iFixit/react-commerce
# Google Tag Manager: Use Everywhere

## Overview

Previously we were using the GoogleTagManager only on the troubleshooting pages. Now, because we want to use Piwik Pro which we are loading via GTM, we want GTM too be loaded on basically every page. This moves the code that was used in the troubleshooting pages to its own file. It then also adds that code to every page.

## QA

Make sure the troubleshooting pages piwik still works, and make sure every other page is loading piwik too. Make sure that your `NEXT_PUBLIC_GTM_KEY` is set correctly.

Closes: #1990
